.cvsignore

```diff
@@ -18,3 +18,5 @@ osmy_vigilance
 init_database.sql
 init_derived_data.sql
 
+/.idea/
```

Comment:
Somehow those unrelated changed made it into the pull request. Not a big issue, just wanted to make you aware of it.